py/newsletter.py

```diff
@@ -14,36 +14,38 @@
 # USER VARIABLES
 # greetings
 greet_title = 'HAPPY WEEK * EVERYONE'
+greet_sub = 'Here are some things to get excited about!'
```

Comment:
so these don't get taken out with the parser?

Reply:
They can be changed through the txt file. Those are left there as defaults.